src/main/java/controllers/FeedController.java

```diff
@@ -80,7 +61,8 @@ private void runPostAction(Post selectedPost, Integer postAction) {
      */
     private String generateDisplayedPosts(ArrayList<Post> posts) {
         this.postsActionMap = new HashMap<>();
-        StringBuilder postsString = new StringBuilder("Enter a post number for a detailed view of that post: ");
+        StringBuilder postsString = new StringBuilder("Enter a post number for a detailed view of " +
+                "that post or enter 99 to choose a different feed filter: ");
 
         for (int i = 0; i < posts.size(); i++) {
             this.postsActionMap.put(i, posts.get(i));
```

Comment:
I think this postsString is showing "1's Amazing Recipe" instead of "eric's amazing recipe" so it should show the username instead of userID